src/site/sphinx/client-basics.rst

```diff
@@ -52,6 +52,22 @@ like the following:
         }
     });
 
+You can also use builder pattern for richer client construction:
```

Comment:
the builder pattern.

Also not sure what you mean by richer, maybe we can just leave the adjective out.